charts/jaeger/README.md

```diff
@@ -313,6 +313,7 @@ The following table lists the configurable parameters of the Jaeger chart and th
 | `schema.activeDeadlineSeconds` | Deadline in seconds for cassandra schema creation job to complete | `120` |
 | `schema.keyspace` | Set explicit keyspace name | `nil` |
 | `spark.enabled` | Enables the dependencies job| `false` |
+| `spark.extraEnv` | Additional environment variables | {} |
```

Comment:
@naseemkullah @cgroschupp I think there is a typo, `{}` should be `[]`, as that's what you set as default in [Values.yaml file](https://github.com/jaegertracing/helm-charts/pull/119/files#diff-630d916c964d14d3d12a7a96261e789fR398).

Reply:
good catch @mehta-ankit , this will be addressed in https://github.com/jaegertracing/helm-charts/pull/121, please take a look!